src/librustc_mir/interpret/traits.rs

```diff
@@ -55,7 +55,7 @@ impl<'a, 'mir, 'tcx, M: Machine<'a, 'mir, 'tcx>> InterpretCx<'a, 'mir, 'tcx, M>
         );
         let tcx = &*self.tcx;
 
-        let drop = crate::monomorphize::resolve_drop_in_place(*tcx, ty);
+        let drop = rustc_monomorphize::resolve_drop_in_place(*tcx, ty);
```

Comment:
This, and other `Instance` constructors, should be moved to `Instance` methods in `rustc`, instead.

Reply:
It appears there is _a lot_ of repeated code, so I deduplicated it and move it to methods on `rustc::ty::Instance`